.storybook/preview.tsx

```diff
@@ -1,13 +1,13 @@
-import * as React from "react";
+import * as React from 'react';
 import { IntlProvider } from 'react-intl';
-import { initialize, mswDecorator } from 'msw-storybook-addon';
+import { initialize, mswLoader } from 'msw-storybook-addon';
```

Comment:
mswDecorator is deprecated in favor of mswLoader
https://github.com/mswjs/msw-storybook-addon/blob/main/MIGRATION.md#mswdecorator-is-deprecated-in-favor-of-mswloader